commands/channel.go

```diff
@@ -273,10 +275,6 @@ func archiveChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) err
 }
 
 func listChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	if len(args) < 1 {
```

Comment:
This is removed because the `cobra.MinimumNArgs(1)` will take care of min args command